src/webgl/shaders/light.vert

```diff
@@ -17,6 +17,9 @@ uniform vec3 uPointLightLocation[8];
 uniform vec3 uPointLightColor[8];
 uniform bool uSpecular;
 uniform float uShininess;
+uniform float constAtt;
```

Comment:
I think these should also be `uConstantAttenuation, uLinearAttenuation, uQuadraticAttenuation` prepended with the `u` to keep them consistent with other shader uniforms.